.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  extends: ["react-app", "prettier"],
```

Comment:
Self made rulesett :D ?

Reply:
It is a npm package that: 
_Turns off all rules that are unnecessary or might conflict with Prettier._

Prettier is an extension I use to format the code, however I actually removed it later on in the project and instead format using ESLint instead, so this extended config should be removed.

Reply:
I ment the whole setup, but thanks for the reply :D 
But the setup turns off stuff that conflict with prettier?
Okok

Reply:
Good that you are effficient with linting :D

Reply:
Ah right, the react-app contains multiple "best practice"-rules which I then extend/override with some that are personal preference e.g. no semicolon required and similar ones :)

Reply:
Cool cool!